Gemfile

```diff
@@ -42,7 +42,7 @@ group :development, :test do
   gem "factory_bot_rails", "~> 5"
   gem "govuk-lint", "~> 3"
   gem "govuk_schemas", "~> 3.2"
-  gem "govuk_test", "~> 0.3"
```

Comment:
Any reason why we're doing a more specific pinning?

Reply:
What more specific pinning? ;-)